systems-tools/artifact-rdc.toml

```diff
@@ -0,0 +1,22 @@
+[components.dbg."systems-tools/stage"]
+[components.dev."systems-tools/stage"]
+include = [
+  "include/**",
+  "lib/cmake/**",
+]
+[components.doc."systems-tools/stage"]
+include = [
+  "share/doc/rdc/**",
+]
+[components.lib."systems-tools/stage"]
+include = [
+  "lib/*.so*",
+  "lib/rdc/**",
+]
+[components.run."systems-tools/stage"]
+include = [
+  "lib/*.so*",
+  "lib/rdc/*.so*",
```

Comment:
Those are included in `components.lib`. The artifact `components.run` will depend on `components.lib` and needs to use the libs there. See https://github.com/ROCm/TheRock/blob/main/docs/development/artifacts.md.